packages/entities/entities-shared/src/components/permissions-wrapper/PermissionsWrapper.vue

```diff
@@ -37,4 +37,8 @@ const showSlotContent = computed((): boolean => isAllowed.value === true || (isA
 onBeforeMount(async (): Promise<void> => {
   isAllowed.value = await props.authFunction()
 })
+
+watch(() => props.authFunction, async () => {
+  isAllowed.value = await props.authFunction()
+})
```

Comment:
Are these equivalent to the following?
```suggestion
watch(() => props.authFunction, async () => {
  isAllowed.value = await props.authFunction()
}, { immediate: true })
```